docker-compose.yml

```diff
@@ -0,0 +1,27 @@
+version: '3.8'
+
+services:
+  phoenix:
+    build:
+      context: .
+    environment:
+      POSTGRES_USER: postgres
+      POSTGRES_PASSWORD: postgres
+      POSTGRES_DB: nudge_dev
+      POSTGRES_PORT: 5432
+      POSTGRES_HOST: postgres
+    ports:
+      - "4000:4000"
+    depends_on:
+      - db
+  db:
+    image: postgres:latest
```

Comment:
don't use `latest`. Let's restrict this current version. I believe its 13, but you can check here: https://hub.docker.com/_/postgres?tab=description

Preferably use alpine versions of the image, which uses very little disk space